brave/brave-core
# RPill Implementation

Resolves brave-intl/bat-reputation#192

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:

1. Start the browser on the following platform configurations i) Win and ii) Win running on a VM.
2. Enable Rewards;
3. Request token refill from Ads;
4. Check headers from `GET /v1/confirmation/payment/{payment_id}` request:
- For platform configuration i) there should be an header set as `via: 1.0 brave.com (Apache/2.4.0)`
- For platform configuration ii) there should be an header set as `via: 1.0 brave.com (Apache/2.4.1)`
